syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -156,13 +195,7 @@ def get(self):
             else:
                 shares.append(share)
 
-        res_field = sum(shares) % self.field
-
-        gate = res_field.native_gt(self.field / 2).long()
-        neg_nums = (res_field - self.field) * gate
-        pos_nums = res_field * (1 - gate)
-        result = neg_nums + pos_nums
-
+        result = self.modulo(sum(shares), self.field) if self.dtype == "custom" else sum(shares)
```

Comment:
Maybe add a small comment to say that the modulo is performed automatically with torch.int32 and torch.int64 values (if I'm not wrong ^^)

Reply:
Yes you are right. I will add the comment.